project/plugins.sbt

```diff
@@ -1,17 +1,16 @@
 addDependencyTreePlugin
 
-addSbtPlugin("org.scalameta"     % "sbt-scalafmt"                 % "2.5.2")
-addSbtPlugin("org.scalameta"     % "sbt-mdoc"                     % "2.5.2")
-addSbtPlugin("com.timushev.sbt"  % "sbt-updates"                  % "0.6.4")
-addSbtPlugin("com.github.cb372"  % "sbt-explicit-dependencies"    % "0.3.1")
-addSbtPlugin("org.typelevel"     % "sbt-typelevel-no-publish"     % "0.7.1")
-addSbtPlugin("org.typelevel"     % "sbt-typelevel-mima"           % "0.7.1")
-addSbtPlugin("org.typelevel"     % "sbt-typelevel-versioning"     % "0.7.1")
-addSbtPlugin("org.typelevel"     % "sbt-typelevel-scalafix"       % "0.7.1")
-addSbtPlugin("org.typelevel"     % "sbt-typelevel-settings"       % "0.7.1")
-addSbtPlugin("org.typelevel"     % "sbt-typelevel-github"         % "0.7.1")
-addSbtPlugin("org.typelevel"     % "sbt-typelevel-github-actions" % "0.7.1")
```

Comment:
Those are removed in favor of `sbt-typelevel-ci-release` that includes them both